rust/src/composepost.rs

```diff
@@ -55,6 +58,21 @@ fn postprocess_presets(rootfs_dfd: &openat::Dir) -> Result<()> {
     Ok(())
 }
 
+// We keep hitting issues with the ostree-remount preset not being
+// enabled; let's just do this rather than trying to propagate the
+// preset everywhere.
+fn postprocess_rpm_macro(rootfs_dfd: &openat::Dir) -> Result<()> {
+    let rpm_dir = "usr/etc/rpm";
```

Comment:
Why `/etc` instead of `/usr/lib/rpm/macros.d`?

Reply:
Oh yeah, that's clearly better. Was fixated on `/etc` because that's what I was using in testing.